catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/SeedCommandTest.java

```diff
@@ -205,6 +212,19 @@ public void testDoesNotDownloadCachedResource() throws Exception {
         verify(catalogFramework, times(2)).query(any(QueryRequest.class));
     }
 
+    @Test
+    public void testDoesNotSkipResources() throws Exception {
+
+        int maxPageSize = NumberUtils.toInt(System.getProperty("catalog.maxPageSize"), 1000);
+        int resourceLimit = maxPageSize * 2;
```

Comment:
The resource limit should be higher than the max page size to prove the test is valid. So it doesn't necessarily need to be double, but I felt it was a simple way to convey that. The JavaDoc I'm writing will explain this